CHANGES.rst

```diff
@@ -93,7 +93,8 @@ Bugs fixed
   Set this option to ``False`` to report HTTP 401 (unauthorized) server
   responses as broken.
   Patch by James Addison.
-* #11868: linkcheck: added a distinct ``timeout`` reporting status code.
+* #11868: linkcheck: added a distinct ``timeout`` reporting status code,
+  optionally enabled by setting ``linkcheck_discriminate_timeout`` to ``True``.
```

Comment:
```suggestion
  optionally enabled by setting :confval:`linkcheck_discriminate_timeout` to ``True``.
```

Reply:
Ah, thank you.  I should fix `linkcheck_allow_unauthorized` in `CHANGES.rst` too.